### ISSUE TITLE
Error and Warning with #criu check command

### ISSUE THREAD

**Issue by rowshan** [checkpoint-restore/criu#493]
Hi, 
I am trying to use criu for live container migration. I run the following  `` sudo criu check`` command and got the following errors and warning:

``sro@sro-thinkpad:~/Desktop/criu$ sudo criu check
Warn  (criu/kerndat.c:808): Stale /run/criu.kdat file
Warn  (criu/net.c:2770): Unable to get socket network namespace
Warn  (criu/net.c:2770): Unable to get tun network namespace
Warn  (criu/sk-unix.c:214): sk unix: Unable to open a socket file: Bad address
Warn  (criu/libnetlink.c:55): ERROR -95 reported by netlink
Error (criu/net.c:3085): Unable to create a veth pair: -95
Warn  (criu/net.c:3107): NSID isn't reported for network links
Warn  (criu/net.c:2770): Unable to get socket network namespace
Warn  (criu/autofs.c:79): Failed to find pipe_ino option (old kernel?)
Looks good.
 ``

please let me know how to solve this problem. I have ubuntu 16.04.2 machine .

**Comment by rppt** [comment]
What is your kernel version?

**Comment by rst0git** [comment]
and what is the criu version?

**Comment by farooqte96** [comment]
Can you run criu check after updating your kernel?

**Comment by rppt** [comment]
@rowshan, 4.4 is more than two years old, some features that CRIU may use in certain circumstances were added after 4.4 was released.
Core functionality will work, but with older kernels CRIU is has more restrictions on what can be migrated.
You can try installing newer kernel with
```
apt-get -y install linux-image-virtual-hwe-16.04-edge
```

**Comment by rowshan** [comment]
@rppt thanks a lot !

**Comment by rowshan** [comment]
This problem is solved! Thanks a lot
